modules/client/src/multisig/internal/client/methods.ts

```diff
@@ -372,6 +375,9 @@ export class MultisigClientMethods extends ClientCore
     if (!proposalId) {
       throw new InvalidProposalIdError();
     }
+    if (!/^0x[A-Fa-f0-9]{40}_(0x[A-Fa-f0-9]{1,64})$/.test(proposalId)) {
```

Comment:
Use the common function to validate

Reply:
the common function doesn't validate but also returns stuff. Idk if this is the expected way how to test it. @brickpop what shall I do?

Reply:
@mathewmeconry `isProposalId()` should be returning `true/false`, so encapsulating the test there sounds simpler than repeating the code in some places. `decodeProposalId()` returns the values, but I don't think we need this here

Reply:
I removed the check because it was duplicate

Reply:
I still see the two if's. I suggest we replace the "if" blocks by this one:

```ts
if (!isProposalId(proposalId)) { ... }
```

Reply:
missed this one. it is fixed now